android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativePackage.java

```diff
@@ -26,7 +27,8 @@
     private InstabugColorTheme instabugColorTheme = InstabugColorTheme.InstabugColorThemeLight;
 
     public RNInstabugReactnativePackage(String androidApplicationToken, Application androidApplication,
-                                        String invocationEventValue, String primaryColor) {
+                                        String invocationEventValue, String primaryColor,
+                                        InstabugFloatingButtonEdge floatingButtonEdge, int offset) {
```

Comment:
@salmatarzi This will break users' apps, we should deprecate the old constructor and change its implementation to make use of this constructor and not to duplicate the code.